src/event_store_ms/src/main/java/database/DatabaseHandler.java

```diff
@@ -0,0 +1,240 @@
+package database;
+
+import DTO.HotelDTO;
+import DTO.ReservationDTO;
+import DTO.TransportDTO;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import events.HotelEvent;
+import events.ReservationEvent;
+import events.TransportEvent;
+
+import java.sql.*;
+
+public class DatabaseHandler {
+
+    private Connection conn;
+
+    public DatabaseHandler(Connection conn) {
+        this.conn = conn;
+    }
+
+    public boolean saveReservationEvent(ReservationEvent reservationEvent) {
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
+            objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+            String json = objectMapper.writeValueAsString(reservationEvent);
+
+            String type = reservationEvent.getTitle();
+            String operation = reservationEvent.getReservation_status();
+            String from = "ReservationMS";
+            Timestamp receiveDate = new Timestamp(System.currentTimeMillis());
+
+            String sql = "INSERT INTO eventslog (Type, Operation, \"From\", ReceiveDate, Body) VALUES (?, ?, ?, ?, ?::json)";
+            PreparedStatement stmt = conn.prepareStatement(sql);
+            stmt.setString(1, type);
+            stmt.setString(2, operation);
+            stmt.setString(3, from);
+            stmt.setTimestamp(4, receiveDate);
+            stmt.setString(5, json);
+
+            int rowsAffected = stmt.executeUpdate();
+            System.out.println(rowsAffected + " rows inserted");
+            stmt.close();
+
+            return true;
```

Comment:
Nazwa metody totalnie nie indykuje że zwraca boola, trochę dziwne jest też to że ukrywamy ten error return false'em. Ja bym wywalił tego true/false i jak coś się wywali to łapiemy error w miejscu wywołania funkcji a nie tutaj